erpnext/accounts/doctype/account/test_account.py

```diff
@@ -241,6 +241,28 @@ def test_child_company_account_rename_sync(self):
 		for doc in to_delete:
 			frappe.delete_doc("Account", doc)
 
+	def test_validate_account_currency(self):
```

Comment:
Correct me if I'm wrong, but it seems like this test does not cover the problem described in the issue.